deploy/charts/kubernetes-cluster-prep/templates/golden_configmap.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.authnK8s.configMap.create -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{ .Values.authnK8s.configMap.name }}
+    labels:
+      release: {{ .Release.Name }}
+      heritage: {{ .Release.Service }}
+      app.kubernetes.io/name: {{ .Values.authnK8s.configMap.name }}
+      app.kubernetes.io/component: "conjur-config"
+      app.kubernetes.io/instance: {{ .Release.Namespace }}
+      app.kubernetes.io/part-of: "authn-k8s-cluster-config"
+      helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+data:
+    # authn-k8s Configuration
```

Comment:
I've been making changes to get unittest to work. Probably easiest to lift the rest of this configmap data section from here:
https://github.com/diverdane/conjur-cluster-prep/blob/master/templates/golden_configmap.yaml#L7